index.html

```diff
@@ -1282,7 +1308,17 @@ <h4>PgpVerificationKey2021</h4>
 
       <section>
         <h4>RsaVerificationKey2018</h4>
-
+        <p class="issue">
```

Comment:
Just for future information, ReSpec has built in tooling to reference Github issues, we should be using those features instead of hand-coding URLs.

Reply:
Thanks, I'll take a look at the documents to figure out how that was done. I did it this way because that's how I saw another issue was doing it, but I think it was only happening because it was referencing an external repo issue.